.github/workflows/release.yml

```diff
@@ -0,0 +1,59 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+name: Handle Release
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+        
+      - name: Set GOPATH
+      # temporary fix
+      # see https://github.com/actions/setup-go/issues/14
+        run: |
+          echo "##[set-env name=GOPATH;]$(dirname $GITHUB_WORKSPACE)"
+          echo "##[add-path]$(dirname $GITHUB_WORKSPACE)/bin"
```

Comment:
Where does it come from?

https://stackoverflow.com/questions/59236611/github-actions-run-golint-on-wihdows/59242962#59242962

Reply:
GitHub actions have some issues in env variables, so there are some solutions wrote by GitHub community, I wrote one of them, your option is another good option. Those are temporary solutions until GitHub takes out a new way to do it or get a solution.

Reply:
It's only GOPATH